### PR TITLE
Collect global Envvars

### PR DESCRIPTION
**Pull request by kobergj**
Compose a list of envvars living in more than 1 service

### CODE REVIEW

**Comment by mmattel** [comment]
We need to change in the adoc template file two things:
* `[width="100%",cols="30%,20%,~,~,~",options="header"]`
the first two columns need a fixed width
* for the xref column only, `a| [subs=attributes+]`
`-attributes` --> `attributes+`

image

Then, as you can see, we need, if possible two sortings 🙈 
* all services listed should be sorted alphabetical ascending
see in the image above that the `app-provider` service is "somewhere" in the middle and therefore hard to identify. the list is usually looong.
* if possible, the names of the envvars should be sorted ascending

**Comment by kobergj** [comment]
@mmattel yes good point :+1:  I was not 100% what you mean by
```
if possible, the names of the envvars should be sorted ascending
```
so I sorted them alphabetically. Should I sort by number of services?
Rest is implemented as requested.

File: docs/templates/ADOC_global.tmpl [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[caption=]
+.Environment variables with global scope available in multiple services
+[width="100%",cols="30%,20%,~,~,~",options="header"]
```
Comment (mmattel) [inline docs/templates/ADOC_global.tmpl:3]:
```suggestion
// collected through docs/helpers/adoc-generator.go.tmpl

[.landscape]
[caption=]
.Environment variables with global scope available in multiple services
[width="100%",cols="30%,25%,~,~,~",options="header"]
```

File: docs/templates/ADOC_global.tmpl [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| Default Value
+| Description
+
+{{ range . }}
+
+a| `{{ .Name }}`
+
+a| [subs=attributes+]
+{{- range .Services}}
+xref:{s-path}/{{ . }}.adoc[{{ . }}] +
```
Comment (mmattel) [inline docs/templates/ADOC_global.tmpl:17]:
```suggestion
* xref:{s-path}/{{ . }}.adoc[{{ . }}] +
```
When adding a * it makes a list item out of it. Then readability improves a lot when rendered.

File: docs/templates/ADOC_global.tmpl [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+{{- end }}
+
+a| [subs=-attributes]
+++{{ .Type }} ++
+
+a| [subs=-attributes]
+++{{ .DefaultValue }} ++
+
+a| [subs=-attributes]
+++{{ .Description }} ++
```
Comment (mmattel) [inline docs/templates/ADOC_global.tmpl:27]:
```suggestion
{{ .Description }}
```
No need to adoc escape here, now 1:1 as in ADOC.tmpl

**Review by mmattel — APPROVED** [review]
LGTM, a build and check on staging looks perfect 😄 👍

**Comment by mmattel** [comment]
When merged, I will backport the global adoc file to the stable docs branch.

**Pull request merged**